app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  before_action :logged_in_user
+
+  def create
+    @user = User.find(params[:followed_id])
+    current_user.follow(@user)
```

Comment:
Note to research: So sánh scope và class method

Reply:
dạ a